### ISSUE TITLE
Normal user copy of NVDA sometimes speaks when in a secure desktop

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2163]
**Reported by k_kolev1985 on 2012-03-13 20:50**
Hello, 

When I put my Windows 7 Ultimate SP1 32-bit machine in StandBy mode and then wake it up, there's a double talk from NVDA in the Windows logon screen. The first voice is the one set for the logon screen, and the other is from my Windows session. The one from my windows session doesn't read the logon screen itself, but the output from the "nvdaControllerClient32.dll", witch is used by "SkypeTalking". The voice from the Windows session can't be interrupted by pressing the CONTROL key. Is this normal behavior? Using the latest main snapshot.

**Comment by bhavyashah** [comment]
If I am on the same page here, I am fairly certain that this issue still occurs sometimes with Windows 8.1 and NVDA 2017.2. While the implementation cost of fixing such a bug is unknown to me, I would suggest assigning a P3 or P4.

**Comment by bhavyashah** [comment]
Status update: I can confirm that this issue persists with NVDA version 2020.2 Beta 1 on a Windows 8.1 system. I am unable to give specific steps to reproduce at the moment, but I certainly recall experiencing this double speaking from the Secure Screen NVDA voice and the Desktop NVDA voice in the recent past. Both voices are different so I am able to distinguish between Secure Screen NVDA and Desktop NVDA.

**Comment by Adriani90** [comment]
Closing in favor of #7626 which is basically the same problem.
